x-pack/plugins/alerting/server/task_runner/types.ts

```diff
@@ -140,19 +140,25 @@ export type Executable<
 
 export interface RuleTypeRunnerContext {
   alertingEventLogger: AlertingEventLogger;
-  flappingSettings: RulesSettingsFlappingProperties;
+  flappingSettings?: RulesSettingsFlappingProperties;
   namespace?: string;
-  queryDelaySettings: RulesSettingsQueryDelayProperties;
+  queryDelaySec?: number;
   ruleId: string;
   ruleLogPrefix: string;
   ruleRunMetricsStore: RuleRunMetricsStore;
   spaceId: string;
 }
 
+export interface RuleRunnerErrorStackTraceLog {
```

Comment:
I'm sure I just saw this defined in a different module.  Seems fine though - for such small types, not sure reuse is worth the hassle.